CHANGELOG.md

```diff
@@ -10,7 +10,12 @@
 
 #### Enhancements
 
-* None.
+* Add `contains_over_range_not_nil` rule to prefer using `contains` over comparison of `range(of:)` to `nil`.  
```

Comment:
same as the other PR, can you please mention that this is an opt-in rule and add `#` before the issue number?